app/views/account/pages/_rate_buttons.html.slim

```diff
@@ -0,0 +1,8 @@
+button onclick="changeRating(+1000);" class='btn btn-success btn-sm' +1000
```

Comment:
what if tomorrow I want to add new option "+88", how can I do it? edit in 2 places?
Use LEGAL_RATING variable and iterate over IT. DRY you code